ykcapi/src/lib.rs

```diff
@@ -25,7 +25,7 @@ use yksmp::{Location as SMLocation, StackMapParser};
 
 #[no_mangle]
 pub extern "C" fn yk_mt_new() -> *mut MT {
-    let mt = Box::new(MT::new());
+    let mt = Box::new(MT::new().unwrap());
```

Comment:
I suppose at some stage this error should be passed up to C code as a null pointer?

Reply:
See the next commit.